src/ui/src/app/deployment-item/deployment-item.component.html

```diff
@@ -0,0 +1,20 @@
+<a class="deployment_item" href="/deployments/{{ deployment.id }}" routerLink="/deployments/{{ deployment.id }}"
+  [style.background-color]="backgroundColor"
```

Comment:
Same here about the issue of set a backgroundColor based on the icon color. We don't control it.